### ISSUE TITLE
A Critical Analysis of Linux Landscape (2023) | Animesh Sahu

### ISSUE THREAD

**Issue by Animeshz** [Animeshz/site#10]
https://animeshz.github.io/site/blogs/a-critical-analysis-of-linux-landscape.html? 

 Is the Linux Ecosystem Heading in the Right Direction?

**Comment by BillDietrich** [comment]
> If system has went down a wrong path, with all the directory structure and things almost about to burst.

This sentence makes no sense, maybe is missing some words ?

**Comment by BillDietrich** [comment]
To me, the major point in the article is "fragmentation".  That is what is hurting desktop Linux.  Filesystem standard is an internal detail.  Systemd is a good thing (does one thing well: manage units of work), not a bad thing.  Package management problems are another example of fragmentation.  Containers/appbundles are a good thing (decouple app update from system update, get some easy permission control).

How to address fragmentation: persuade leaders of major projects that it's a problem.  Get them to encourage consolidation.  For example, all of the Ubuntu* family should be just options in one installer, not separate distros.  Same for Debian family, RH family, Arch family.  Get all members of each family to share one source tree, one bug-reporting tree.

**Comment by Animeshz** [comment]
> @BillDietrich
> > If system has went down a wrong path, with all the directory structure and things almost about to burst.
> 
> This sentence makes no sense, maybe is missing some words ?

Yes, my English is not complete in that part hence a bit of obstruction in the explanation, apologies about that. A complete case against it is present here: http://kmkeen.com/maintainers-matter/ (HN: https://news.ycombinator.com/item?id=11911303) :)
